modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize and configure an existing project access roles name by editing the YAML code, and add an `console.openshift.io/display-name: Display name` annotation to the cluster.
+
+.Procedure
+To customize and configure the cluster roles of all the projects:
```

Comment:
I would recommend finetuning this here a little bit. The admin will customize the options in the tab here, not the cluster roles themselves. What about this change?

```suggestion
To customize which cluster roles options are shown for all projects in the *Project Access* tab:
```

Reply:
Great suggestion, I am gonna make this change.